packages/ember-htmlbars/lib/system/build-component-template.js

```diff
@@ -186,7 +186,7 @@ function normalizeComponentAttributes(component, attrs) {
     }
   }
 
-  normalized.role = buildStatement('get', 'ariaRole');
+  normalized.role = normalized.role || buildStatement('get', 'ariaRole');
```

Comment:
Hmm. I see that this is what @krisselden suggested in #14007. 

Seems good to me.